pkg/block/gs/adapter.go

```diff
@@ -7,7 +7,6 @@ import (
 	"io"
```

Comment:
We should also test that this doesn't break Multipart Upload REST API(LakeCTL)

Reply:
LakeCTL doesn't do multipart upload
That leaves the REST API for multipart- I read the code- seems like a low risk that anything in this change would break it- not sure it's worth testing